.buildkite/pipeline.yml

```diff
@@ -38,19 +38,14 @@ steps:
     env: *common_env
     plugins: *common_plugins
 
-  - label: ':swift: SwiftLint'
-    key: swiftlint
-    command: .buildkite/commands/swiftlint.sh --strict
-    agents:
-      queue: default
-    plugins:
-      - docker#v5.8.0:
-          # TODO: Ensure the same version as local is used, i.e. with Buildkite env vars injection during pipeline upload
-          image: ghcr.io/realm/swiftlint:latest
-          mount-buildkite-agent: true # Needed for calling `buildkite-agent annotate`
+  - label: ":swift: SwiftLint"
+    command: run_swiftlint --strict
+    plugins: *common_plugins
     notify:
       - github_commit_status:
-          context: SwiftLint
+          context: "SwiftLint"
```

Comment:
I assume this came via copy-paste from the other projects that use the same step. Just wanted to point out that the `""` are redundant. 😄